.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+index.html
```

Comment:
might be a good idea to exclude the css and images that were copied in

Reply:
I will now as they'll be embeded in the HTML